samples/client/petstore/swift5/alamofireLibrary/SwaggerClientTests/SwaggerClientTests/FileUtils.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FileUtils.swift
+//  SwaggerClientTests
+//
+//  Created by Bruno Coelho on 11/03/2020.
+//  Copyright © 2020 Swagger. All rights reserved.
```

Comment:
Please use `Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)` instead.

Reply:
Like this?

Reply:
Yup